code-experiments/src/suite_toy.c

```diff
@@ -25,8 +25,9 @@ static coco_suite_t *suite_toy_initialize(void) {
 
   coco_suite_t *suite;
   const size_t dimensions[] = { 2, 3, 5, 10, 20 };
+  const size_t num_dimensions = sizeof(dimensions) / sizeof(dimensions[0]);
 
-  suite = coco_suite_allocate("toy", 6, 3, dimensions, "instances: 1");
+  suite = coco_suite_allocate("toy", 6, num_dimensions, dimensions, "instances: 1");
```

Comment:
@ttusar are you sure that this change is correct? Did the toy suite change?

Reply:
OK, I guess I see now that this just changes the suite to contain different dimensions.